From b2e62adb1ed94f9a3f681bd6d0b8117b2ec3c1f9 Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@src.gnome.org>
Date: Sun, 14 Mar 2010 21:24:21 +0000
Subject: Replaced deprecated macros -- GTK_WIDGET_REALIZED & GTK_WIDGET_VISIBLE

These macros have been deprecated since Gtk+ 2.20, and their
replacements were introduced in Gtk+ 2.18. Thus, the minimum Gtk+
version has been bumped accordingly.
---
diff --git a/configure.ac b/configure.ac
index 564c0b0..0189b88 100644
--- a/configure.ac
+++ b/configure.ac
@@ -23,7 +23,7 @@ AC_SUBST(GNOTE_VERSION)
 dnl all the library version.
 dnl if one is harcoded elsewhere, it is a bug
 
-GTK_VERSION=2.12
+GTK_VERSION=2.18
 LIBGLIBMM_VERSION=2.0
 LIBGTKMM_VERSION=2.12.0
 LIBGCONF_VERSION=2.0.0
diff --git a/libtomboy/tomboyutil.c b/libtomboy/tomboyutil.c
index 162e4a2..9de7e62 100644
--- a/libtomboy/tomboyutil.c
+++ b/libtomboy/tomboyutil.c
@@ -153,9 +153,9 @@ tomboy_window_override_user_time (GtkWindow *window)
 void
 tomboy_window_present_hardcore (GtkWindow *window)
 {
-	if (!GTK_WIDGET_REALIZED (window))
+	if (!gtk_widget_get_realized (GTK_WIDGET (window)))
 		gtk_widget_realize (GTK_WIDGET (window));
-	else if (GTK_WIDGET_VISIBLE (window))
+	else if (gtk_widget_get_visible (GTK_WIDGET (window)))
 		tomboy_window_move_to_current_workspace (window);
 
 	tomboy_window_override_user_time (window);
--
cgit v0.8.3.1
